forge/forge/llm/providers/groq.py

```diff
@@ -111,24 +87,24 @@ def get_api_access_kwargs(self) -> dict[str, str]:
 
 
 class GroqSettings(ModelProviderSettings):
-    configuration: GroqConfiguration  # type: ignore
     credentials: Optional[GroqCredentials]  # type: ignore
     budget: ModelProviderBudget  # type: ignore
 
 
-class GroqProvider(BaseChatModelProvider[GroqModelName, GroqSettings]):
+class GroqProvider(BaseOpenAIChatProvider[GroqModelName, GroqSettings]):
+    CHAT_MODELS = GROQ_CHAT_MODELS
+    MODELS = CHAT_MODELS
+
     default_settings = GroqSettings(
         name="groq_provider",
         description="Provides access to Groq's API.",
-        configuration=GroqConfiguration(
-            retries_per_request=7,
-        ),
+        configuration=ModelProviderConfiguration(),
         credentials=None,
         budget=ModelProviderBudget(),
     )
 
     _settings: GroqSettings
-    _configuration: GroqConfiguration
+    _configuration: ModelProviderConfiguration
```

Comment:
Should this still be a GroqConfig that inherits? repeat for all

Reply:
IMO don't create classes that don't add any functionality. If the base class suffices, just use that. Also makes for less type errors.